src/App.scss

```
@@ -1 +1,39 @@
-// style go here
+@import url("https://fonts.googleapis.com/css2?family=Lilita+One&display=swap");
+
+* {
+  box-sizing: border-box;
+  font-family: "Lilita One", cursive;
+}
```

only `box-sizing` property allows for `*` property
```suggestion
* {
  box-sizing: border-box;
}

html {
  font-family: "Lilita One", cursive;
}
```